docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
+| `services`<br><br>*External*: `Name`<br>*Internal*: `service_id`| Each record defines a transit service and its properties.<br><br>While there are a small number of exceptions (e.g. Solano Express, which is jointly managed by Solano and Napa), generally each transit service is managed by a single organization. Transit services are differentiated from each other by variation (or the potentiality of variation) in one or more of the following:<ul><li>operator `organization` such as City Staff vs a contracted service,<li>`rider requirements` such as ADA Paratransit eligibility, senior status, school trips, etc.</li><li>Operational characteristics such as reservations, on-demand vs fixed-route, service frequencies, and mode</li>Business processes such as transit technology stacks or personnel</li><li>Funding type, which can effect longevity of the service and how well integrated it is with other services managed by the same organization.</li><li>Rider-facing branding, which is often an indicator of one of the above.</li></ul><br>Services MAY:<ul><li>be *reflected by* a `gtfs service data` record</li><li>be *subject to* one or more `rider requirements`</li><li>be *funded by* one or more `funding programs`</li><li>*operated by* one or more `organizations`</li><li>*managed by* one or more organizations</li><li>be *operated in* one or more `place geography`</li><li>*use* one or more `products`. |
+| `gtfs datasets`<br><br>*External*: `gtfs_dataset_id`  | Each record represents a gtfs dataset (feed) that is either a type of GTFS Schedule, Trip Updates, Vehicle Locations or Alerts.  A gtfs dataset MAY:<ul><li>be *disaggregated into* one or more `gtfs service data` records.</li><li>be *produced* by one or more `organizations`</li><li>be *published* by an `organizations`. |
+| `contracts`<br><br> *External*: `Name`<br>*Internal*: `contract_id` | Each record is a contract between `organizations` to either provide one or more `products` or operate one or more `services`.  Each contract has propreties such as execution date, expiration, and renewals.  |
+| `gtfs service data` <br><br> `Name`: Combination of a set of `Services` record(s) and a single `gtfs datasets` record. | Each record links together a single `gtfs dataset` and one (if possible) or more `services`.  Additional fields define how to isolate the service within the `gtfs dataset`.  <br><br>Many services have more than one GTFS dataset which describes their service. Often these are either precursors to *final* datasets (e.g. AC Transit's GTFS dataset is a precursor to the Bay Area 511 dataset) or artifacts produced in other processes such as creating GTFS Realtime (e.g. [the VCTC GTFS produced by GMV](https://airtable.com/appPnJWrQ7ui4UmIl/tblnVt5FZ2FZmDjDx/viw3NtcDP3Qm0BYyG/recJhrNj21mYVETJG?blocks=hide)). The property `Category` indicates if this is the `primary` dataset, a `precursor` or `unknown` in order to distinguish which should be used. |
```

Comment:
Long term recommendation to drop the `gtfs` prefix from this table name to be able to potentially accommodate other dataset types such as GBFS.

```suggestion
| `service-dataset linkage` <br><br> `Name`: Combination of a set of `Services` record(s) and a single `gtfs datasets` record. | Each record links together a single `gtfs dataset` and one (if possible) or more `services`.  Additional fields define how to isolate the service within the `gtfs dataset`.  <br><br>Many services have more than one GTFS dataset which describes their service. Often these are either precursors to *final* datasets (e.g. AC Transit's GTFS dataset is a precursor to the Bay Area 511 dataset) or artifacts produced in other processes such as creating GTFS Realtime (e.g. [the VCTC GTFS produced by GMV](https://airtable.com/appPnJWrQ7ui4UmIl/tblnVt5FZ2FZmDjDx/viw3NtcDP3Qm0BYyG/recJhrNj21mYVETJG?blocks=hide)). The property `Category` indicates if this is the `primary` dataset, a `precursor` or `unknown` in order to distinguish which should be used. |
```